File: program/find-average-of-numbers/find_average_of_numbers.php
<?php
$input = readline("Enter numbers seperated by a space: ");
$arr = explode(" ", $input);
$sumOfNums = 0;
for ($i = 0; $i <= count($arr); $i++) {
    $sumOfNums += $arr[$i];
}
$average = $sumOfNums / count($arr);
echo "The average is $average";
?>

File: program/find-average-of-numbers/FindAverageOfNumbers.kt
fun main() {
    print("Input : ")
    println()
    var input = readLine()

    println("Average of number $input is : " + FindAverageOfNumbers(input!!))

}

private fun FindAverageOfNumbers(input: String): Double {
    val list = input.toCharArray().map(Character::getNumericValue)

    val avg = list.average()

    return avg
}